hyperon_das_atomdb_cpp/src/type_aliases.h

```diff
@@ -0,0 +1,60 @@
+
+/**
+ * @file type_aliases.h
+ * @brief This header file contains type aliases to improve code readability and maintainability
+ *        within the atomdb namespace.
+ *
+ * The type aliases defined in this file are intended to simplify the usage of commonly used
+ * STL containers and types, making the code more concise and easier to understand. The aliases
+ * cover a range of types including optional values, sets, maps, and lists.
+ *
+ * @note 1. The alias MapOfAny for std::unordered_map<std::string, std::any> is commented out due
+ *          to poor performance. It is kept in the file as a reminder of the performance implications.
+ *       2. ListOfAny is an alias for std::vector<std::any>, representing a list of any type.
+ *          Note that while std::vector<std::any> can be useful in some cases, its performance
+ *          should be tested for each specific use case.
+ */
+#pragma once
+
+#include <any>
+#include <memory>
+#include <optional>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+using namespace std;
+
+namespace atomdb {
+
+// Type aliases for readability
+using uchar = unsigned char;
+
+template <typename T>
+using opt = optional<T>;
```

Comment:
It's just there to allow me to abbreviate `optional` with `opt` when coding... call me lazy, fair enough :smile:
But I now agree that `opt` sometimes gets overlooked because it's too short and it caused me some confusion.
I will remove the type alias and replace `opt` with `optional`.

Reply:
Done.